client/pages/Info/Info.jsx

```diff
@@ -1,33 +1,42 @@
 import React from 'react';
+import { InfoMap } from '../../components/InfoMap/InfoMap';
```

Comment:
Not sure what is the problem but map will work properly if you import it with other name (e.g. InfoMap2)

Reply:
@io-design without {} it will import `GoogleApiWrapper` since it's `exported by default`